aip/aog/3001.md

```diff
@@ -1,7 +1,7 @@
 ---
 aip:
   id: 3001
-  state: draft
+  state: aog-reviewing
```

Comment:
Just use `reviewing`.

Reply:
Don't we need some way to distinguish AIPs in the "Reviewing" state from those in the "AoG Reviewing" state?  If not, should AIP-3001 be rewritten not to define the "AoG Reviewing" state?

Reply:
See https://github.com/googleapis/aip/pull/326#issuecomment-540893947. Short answer: No, we do not need that state.